tinygrad/codegen/cstyle.py

```diff
@@ -119,11 +119,11 @@ def kk(s): kernel.append("  "*depth+s)
     elif uop == UOps.LOAD and newvar is not None:
       # TODO: merge with CONST?
       if bufs[args.i] is not None and isinstance(bufs[args.i].realized, RawConst):
-        assert newvar.dtype == dtypes.float, "const can't be float4"
+        assert newvar.dtype == dtypes.float32, "const can't be float4"
         x = bufs[args.i].realized._buf
         if math.isnan(x): val = "NAN"
         elif math.isinf(x): val = ("-" if x < 0 else "") + "INFINITY"
-        else: val = f"{x}" +  ("f" if not dtypes.is_int(bufs[args.i].dtype) else "")
+        else: val = f"{x}" +  ("f" if not dtypes.is_signed_int(bufs[args.i].dtype) else "")
```

Comment:
this might also have been wrongly checked. Assume it should have checked for all int types and not just signed?